packages/component/tsconfig.json

```diff
@@ -1,18 +1,18 @@
 {
   "extends": "../../tsconfig.json",
-  "include": [
-    "./src/**/*.ts",
-    "./src/**/*.tsx",
-    "./src/**/*.json",
-    "../workspace/src",
-    "../../apps/electron/layers/**/src"
+  "exclude": [
+    "lib",
+    "./src/components/page-list/filter/shared-types.tsx",
+    "./src/components/page-list/filter/logic/custom-type.ts",
+    "./src/components/page-list/filter/logic/matcher.ts",
+    "./src/components/page-list/filter/logic/typesystem.ts"
   ],
```

Comment:
does this mean page-list is  actually a `@affine/workspace` component?

Reply:
No, it's also used by `@affine/component` itself